src/main/java/codeu/model/data/User.java

```diff
@@ -41,6 +42,22 @@ public User(UUID id, String name, Instant creation, String hashedPassword, Strin
     this.about = about;
   }
 
+  /**
+   * Constructs a new User to be used during Registration with default about information
+   *
+   * @param id the ID of this User
+   * @param name the username of this User
+   * @param creation the creation time of this User
+   * @param password the password of this User
+   */
+  public User(UUID id, String name, Instant creation, String hashedPassword) {
+    this.id = id;
+    this.name = name;
+    this.creation = creation;
+    this.hashedPassword = hashedPassword;
+    this.about = "about";
```

Comment:
Should this just be an empty string, maybe? We want users' profiles to look okay if they haven't modified this I think.